reviews/2022-metal3-io-annual.md

```diff
@@ -0,0 +1,262 @@
+	- Release process for BMO [#1150](https://github.com/metal3-io/baremetal-operator/pull/1150)
+	- HardwareData custom resource for host inspection data [#1099](https://github.com/metal3-io/baremetal-operator/pull/1099)
+	- PreprovisioningImage API and integration [#936](https://github.com/metal3-io/baremetal-operator/pull/936)
+	- New API and integration of hostFirmwareSettings and firmwareSchema resources [#901](https://github.com/metal3-io/baremetal-operator/pull/901/), [#938](https://github.com/metal3-io/baremetal-operator/pull/938)
+	- Enable RAID for Redfish-based iDRAC driver flavor [#1095](https://github.com/metal3-io/baremetal-operator/pull/1095)
+	- Reboot API Implementation [#424](https://github.com/metal3-io/baremetal-operator/pull/424)
+	- Add physicalDisks and controller parameters to HardwareRAID [#1062](https://github.com/metal3-io/baremetal-operator/pull/1062)
+	- Implement explicit reboot mode options [#795](https://github.com/metal3-io/baremetal-operator/pull/795)
+	- Add live-iso support [#759](https://github.com/metal3-io/baremetal-operator/pull/759)
+	- Add support for detached annotation [#827](https://github.com/metal3-io/baremetal-operator/pull/827)
+	- Add support for ironic custom deploy interface [#884](https://github.com/metal3-io/baremetal-operator/pull/884)
+
+- **ip-address-manager**
+  -  It's a comprehensive tool to manage static IP address allocations in
+     Cluster API Provider Metal3. It has its own controller and quite a few new
+     features are added in this repo.
+
+ **NOTE**
+`A few of the above mentioned features is a result of work done not only in the previous year but before that time period also. We have put it here to highlight these since we didn't put them in the previous review.`
+
+### General Improvements
+
+  - Added kubesec for security analysis of Kubernetes manifest files.
+  - Added gosec for security analysis of go files.
+  - Enhanced e2e and full stack CI building process.
+  - Bi-weekly triaging of issues.
+  - Restrict interface usage of ironic for better security.
+  - Rename all the ‘master’ branches to ‘main’.
+
+## Future Goals
+
+  - One of the main goals for 2022 was to help adoption of Metal3 thus the
+    Metal3 community has started writing a [User-guide](https://metal3io.netlify.app).
+    The community plans to extend the guide continuously and also provide
+    separate tutorials that will showcase the deployment process on bare metal
+    environment that is as close to a production environment as possible.
+
+  - In 2022 certain aspects of Metal3 security has been improved, but there
+    is more planned specially related to the topics of container security
+    and related to the testing of Metal3 security features during e2e tests.
+
+  - There are improvements done on the CI infrastructure of the Metal3
+    project, but there are more to come in terms of securing and speeding the
+    test processes to help contributors receive feedback faster.
```

Comment:
Really late from me (sorry!), but in a similar vain, there is an effort to run Metal3 CI on Ironic upstream: https://review.opendev.org/c/openstack/ironic/+/863873

Reply:
Added, thanks for the info @dtantsur !